modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/FilterOrderControls.js

```diff
@@ -76,19 +119,109 @@ const FilterOrderControls = ({
 				</ClayManagementToolbar.Item>
 			)}
 
-			{sortingURL && (
+			{showDesignImprovements &&
+				orderDropdownItems &&
+				orderDropdownItems.length > 1 && (
+					<ClayManagementToolbar.Item>
+						<ClayDropDownWithItems
+							items={[
+								...orderDropdownItems.map((item) => {
+									return {
+										...item,
+										onClick: (event) => {
+											onOrderDropdownItemClick(event, {
+												item,
+											});
+										},
+									};
+								}),
+								{type: 'divider'},
+								{
+									active: sortingOrder === 'asc',
+									href:
+										sortingOrder === 'asc'
+											? null
```

Comment:
What happen if you pass an item with `href=null` to `ClayDrpodownWithItems`?

Reply:
[This happens](https://codesandbox.io/s/dropdown-with-null-href-jqvuw?file=/src/App.js), it shows the list item as normal, but it does nothing when you click it. Under the hood it renders a button instead of an anchor, but visually it looks the same.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2FyJFeycRK2DB4c%2Fgiphy.gif&token=nice" data-gitmeme-token="nice"><img src="https://media0.giphy.com/media/yJFeycRK2DB4c/giphy.gif" title="Created by gitme.me with /nice"/></a>